dissect/target/filesystems/zip.py

```diff
@@ -71,9 +71,9 @@ def detect(fh: BinaryIO) -> bool:
         finally:
             fh.seek(offset)
 
-    def get(self, path: str) -> ZipFilesystemEntry:
+    def get(self, path: str, *args, **kwargs) -> ZipFilesystemEntry:
```

Comment:
I think it is nicer to pass the variables explicitly like:
```suggestion
    def get(self, path: str, relentry: FilesystemEntry = None) -> ZipFilesystemEntry:
```